### PR TITLE
chore: update cd for ga

### PR DESCRIPTION
**Pull request by wenytang-ms**

### CODE REVIEW

**Review by tecton — DISMISSED** [review]

**Review by LongOddCode — APPROVED** [review]

File: templates/package.json [hunk 1]
```diff
@@ -33,9 +33,7 @@
         "tab/js/default",
         "tab/ts/default",
         "tab/js/non-sso",
-        "tab/ts/non-sso",
-        "tab/js/m365",
-        "tab/ts/m365"
```
Comment (kuojianlu) [inline templates/package.json:38]: Why to remove m365 tab templates?

Comment (wenytang-ms) [reply templates/package.json]: updated

File: .github/scripts/sync-version.js [hunk 2]
```diff
@@ -42,7 +42,10 @@ function updateFileDeps(file, deps) {
     let fileChange = false;
     for(let [key,value] of Object.entries(deps)){
         if(dep_[key] && semver.prerelease(semver.minVersion(dep_[key]))) {
-            if(!(semver.prerelease(semver.minVersion(dep_[key])).includes("alpha") || semver.prerelease(semver.minVersion(dep_[key])).includes("rc"))){
+            if(!(semver.prerelease(semver.minVersion(dep_[key])).includes("alpha") || semver.prerelease(semver.minVersion(dep_[key])).includes("rc") || semver.prerelease(semver.minVersion(dep_[key])).includes("beta"))){
+                continue;
+            }
+            if(key === "@microsoft/teamsfx" && dep_[key] === "0.6.0-beta.0") {
```
Comment (tecton) [inline .github/scripts/sync-version.js:48]: @kuojianlu The hard-coded value here prevents SDK auto update in m365 tab.

**Review by tecton — DISMISSED** [review]

**Review by kuojianlu — DISMISSED** [review]
The hard-coded check for `@microsoft/teamsfx@0.6.0-beta.0` LGTM.

File: .github/scripts/sync-version.js (hunk 2 above)
Comment (kuojianlu) [reply .github/scripts/sync-version.js:48]: Thanks. I think we will still use `0.6.0-beta.0` until GA release.

**Review by hund030 — APPROVED** [review]

**Review by LongOddCode — DISMISSED** [review]

File: .github/scripts/fxcore-sync-up-version.js [hunk 3]
```diff
@@ -25,9 +25,9 @@ else if(templateVersion.includes("rc")){
     templateConfigFile.version = "0.0.0-rc";
     fse.writeFileSync(templateConfig, JSON.stringify(templateConfigFile, null, 4))
 }
-else if(templateVersion.includes("alpha")) {
-    console.log("sync up template in fx-core as 0.0.0-alpha")
-    templateConfigFile.version = "0.0.0-alpha";
+else if(templateVersion.includes("beta")) {
```
Comment (adashen) [inline .github/scripts/fxcore-sync-up-version.js:28]: can we add more instead of override alpha?

Comment (wenytang-ms) [reply .github/scripts/fxcore-sync-up-version.js:28]: updated, please help review again, thanks

File: .github/workflows/cd.yml [hunk 4]
```diff
@@ -198,10 +227,15 @@ jobs:
...
+      - name: publish alpha release to npm org
+        if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/dev' && github.event.inputs.preid == 'alpha')}}
+        run: |
+          npx lerna publish from-package --dist-tag=alpha --yes --allow-branch dev
+      
       - name: publish beta release to npm org
         if: ${{ github.event_name == 'schedule' || (github.event_name == 'workflow_dispatch' && github.ref == 'refs/heads/ga' && github.event.inputs.preid == 'beta')}}
         run: |
-          npx lerna publish from-package --dist-tag=beta --yes --allow-branch ga
+          npx lerna publish from-package --dist-tag=beta --yes --allow-branch dev
```
Comment (hund030) [inline .github/workflows/cd.yml:238]: should be branch ga?

Comment (wenytang-ms) [reply .github/workflows/cd.yml:238]: thanks

**Review by hund030 — APPROVED** [review]

**Review by kuojianlu — APPROVED** [review]

**Review by tecton — APPROVED** [review]

**Review by LongOddCode — APPROVED** [review]

**Review by adashen — APPROVED** [review]

**Pull request merged**